_posts/2024-08-06-czi-eoss6-announcement.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Chan Zuckerberg Initiative considers scikit-learn an Essential Open Source Software"
+date: August 6, 2024
+categories:
+  - Funding
+tags:
+  - Open Source
+  - Funding
+  - Internship
+  - Diversity
+featured-image: sklearn_czi.png
+
+postauthors:
+  - name: Guillaume Lemaitre
+    website: https://github.com/glemaitre
+    image: guillaume-lemaitre.jpg
+  - name: Lucy Liu
+    website: https://github.com/lucyleeow
+    image: lucyliu.jpeg
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+We are delighted to announce that `scikit-learn` is awarded a grant from
+the [Chan Zuckerberg Initiative (CZI)](https://chanzuckerberg.com/)'s [Essential Open
+Source Software for Science
+(EOSS)](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/)
+program. This grant is funded by [Wellcome Trust](https://wellcome.org/).
+As in previous round, this cycle supports open-source software projects that are
+essential to biomedical research. This is the third time that CZI EOSS supports
+`scikit-learn`. In previous rounds, the following projects were funded:
+
+- [CZI EOSS 1](https://chanzuckerberg.com/eoss/proposals/scikit-learn-maintenance-and-enhancement-for-gradient-boosting/)
+  helped at creating to the
+  [`HistGradientBoostingClassifier`](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html) and
+  [`HistGradientBoostingRegressor`](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html) estimators.
+  These estimators are the equivalent of gradient boosting models implemented in
+  `LightGBM` and `XGBoost`.
+- [CZI EOSS 4](https://chanzuckerberg.com/eoss/proposals/maintenance-extension-of-scikit-learn-machine-learning-in-python/)
+  extended `scikit-learn` to work better with missing values and categorical data in
+  several estimators.
+
+Both grants allowed us to maintain and enhance `scikit-learn` to better serve the
+community.
+
+In this new grant, we will focus on improving the evaluation and inspection of
```

Comment:
Should we move this further up? It seems like telling the reader what this grant is about is the most important thing in this article?

But also, telling people about the amazing hist gradient boosting is important :-/

Reply:
Yep, I had also some doubts about the structure. Let me move the past grants in a section at the end of the post.